db/sqldb/portfolio.go

```diff
@@ -153,32 +158,65 @@ func (d *DB) DeletePortfolio(tx db.Tx, id pacta.PortfolioID) ([]pacta.BlobURI, e
 	return buris, nil
 }
 
-func rowToPortfolio(row rowScanner) (*pacta.Portfolio, error) {
-	p := &pacta.Portfolio{Owner: &pacta.Owner{}, Blob: &pacta.Blob{}}
+type portfolioRow struct {
+	Portfolio                         *pacta.Portfolio
+	PortfolioGroupMembershipGroupID   pacta.PortfolioGroupID
+	PortfolioGroupMembershipCreatedAt time.Time
+}
+
+func rowToPortfolioRow(row rowScanner) (*portfolioRow, error) {
+	p := &portfolioRow{Portfolio: &pacta.Portfolio{Owner: &pacta.Owner{}, Blob: &pacta.Blob{}}}
 	hd := pgtype.Timestamptz{}
+	mid := pgtype.Text{}
+	mca := pgtype.Timestamptz{}
 	err := row.Scan(
-		&p.ID,
-		&p.Owner.ID,
-		&p.Name,
-		&p.Description,
-		&p.CreatedAt,
+		&p.Portfolio.ID,
+		&p.Portfolio.Owner.ID,
+		&p.Portfolio.Name,
+		&p.Portfolio.Description,
+		&p.Portfolio.CreatedAt,
 		&hd,
-		&p.Blob.ID,
-		&p.AdminDebugEnabled,
-		&p.NumberOfRows,
+		&p.Portfolio.Blob.ID,
+		&p.Portfolio.AdminDebugEnabled,
+		&p.Portfolio.NumberOfRows,
```

Comment:
optional: You should be able to implement a custom `Scan` method such that you can just `row.Scan(&p.Portfolio, &mid, &mca)` and it will work. Mostly just syntactic sugar, but cleans things up a bit and allows scanning directly into a Portfolio from multiple places.

Same for PortfolioGroup

Reply:
Oh interesting, I'd not thought of doing that. I'll file a bug to take a look at that. I'd like to have that in the quiver.

Reply:
#84